client/src/components/Overview/AddToCart.jsx

```diff
@@ -1,7 +1,140 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
 
-function AddToCart() {
-  return null;
+// TODO: make Select take all unspecified props from parent
+function Select({
+  label,
+  value,
+  options,
+  defaultSelection,
+  disabled,
+  onChange,
+}) {
```

Comment:
Seems like a lot of props to pass down

Reply:
Not sure how to refactor here. Not all of the props are required in a given instance, but they are all necessary for the functionality that I would like the Select to have.